src/main/java/uk/gov/pay/directdebit/payments/services/GovUkPayEventToPaymentStateMapper.java

```diff
@@ -9,10 +9,12 @@
 import java.util.Optional;
 import java.util.Set;
 
+import static uk.gov.pay.directdebit.events.model.GovUkPayEventType.PAYMENT_CREATED;
 import static uk.gov.pay.directdebit.events.model.GovUkPayEventType.PAYMENT_SUBMITTED;
 
 public class GovUkPayEventToPaymentStateMapper {
     private static final Map<GovUkPayEventType, PaymentState> GOV_UK_PAY_EVENT_TYPE_TO_PAYMENT_STATE = Map.of(
+            PAYMENT_CREATED, PaymentState.CREATED,
```

Comment:
Doesn’t this also require updating `GovUkPayEventStateGraph` to say that `PAYMENT_CREATED` followed by `PAYMENT_SUBMITTED` is a valid sequence of events?

Reply:
I think it makes more sense to do that when I'm implementing the "Make a new payment being inserted into the database save a PAYMENT_CREATED GOV.UK Pay event" acceptance criteria (which will be the next PR)

Reply:
OK!